_posts/2022-01-20-reached-4000-users.md

```diff
@@ -0,0 +1,21 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'Usegalaxy.eu surpassed 40.000 registered users'
+tags: [galaxy, support, society, report]
+supporters:
+- galaxy-europe
+---
+
+We are of celebration! The last November usegalaxy.eu reached 40.000 registered users! Over the last year, the number of registered users has almost doubled, from 25,000 to more than 41,000 by January 2022. As we can see in figure 1, the growth in the number of new registrants shows an almost exponential trend since 2013.
```

Comment:
```suggestion
We are celebrating that usegalaxy.eu reached 40,000 registered users last November! Over the last year, the number of registered users has almost doubled, from 25,000 to more than 41,000 by January 2022. As we can see in figure 1, the growth in the number of new registrants shows an almost exponential trend since 2013.
```